app/utils/importFile.js

```diff
@@ -16,35 +16,55 @@ const importFile = async ({
   documentId,
   collectionId,
 }: Options): Promise<Document> => {
-  return new Promise((resolve, reject) => {
-    const reader = new FileReader();
+  return new Promise(async (resolve, reject) => {
```

Comment:
We could clean this a little if we move the `Promise` creation down to only wrap the file reader. This first flow for server side import can use async/await as there are no callback involved.

Reply:
https://github.com/outline/outline/pull/1469/commits/44a861b3af5b5db96d9a0a15c042bda85c2f2b17#diff-538eabbd5033bdf49a45dce7f55b5545R19